plugins/inputs/snmp/README.md

```diff
@@ -22,8 +22,11 @@ information.
 ```toml
 [[inputs.snmp]]
   ## Agent addresses to retrieve values from.
+  ##   format:  agents = ["<scheme>://<hostname>:<port>"]
```

Comment:
I would mention that `scheme` and `port` are optional.

Reply:
Done, thank you!